README.md

```diff
@@ -3,7 +3,7 @@
 This repo builds on the [r binder](https://github.com/binder-examples/r) and [jupyter lab binder](https://github.com/binder-examples/jupyterlab) and is complementary to the [multi-language-demo binder](https://github.com/binder-examples/multi-language-demo) with examples on using both R and python in both Jupyter Lab and RStudio.
 
  - Launch in Jupyter Lab: [![Binder](http://mybinder.org/badge.svg)](http://beta.mybinder.org/v2/gh/binder-examples/r_with_python/master?urlpath=lab)
- - Launch in RStudio: [![Binder](http://mybinder.org/badge.svg)](http://beta.mybinder.org/v2/gh/KopfLab/binder-examples/r_with_python/master?urlpath=rstudio)
+ - Launch in RStudio: [![Binder](http://mybinder.org/badge.svg)](http://beta.mybinder.org/v2/gh/binder-examples/r_with_python/master?urlpath=rstudio)
```

Comment:
```suggestion
 - Launch in RStudio: [![Binder](http://mybinder.org/badge.svg)](http://mybinder.org/v2/gh/binder-examples/r_with_python/master?urlpath=rstudio)
```

While we are refactoring we can also remove the beta subdomain that we should slowly phase out :)